src/commands/Misc/ship.ts

```diff
@@ -53,8 +53,11 @@ export default class extends SkyraCommand {
 
 	public async run(message: KlasaMessage, [firstUser, secondUser]: [KeyedMemberTag, KeyedMemberTag]) {
 		// We need the UserTags to resolve the avatar and username
-		const firstUserTag = this.client.userTags.get(firstUser.id)!;
-		const secondUserTag = this.client.userTags.get(secondUser.id)!;
+		let firstUserTag = this.client.userTags.get(firstUser.id);
+		let secondUserTag = this.client.userTags.get(secondUser.id);
+
+		if (!firstUserTag) firstUserTag = { avatar: null, discriminator: '0001', bot: false, username: '' };
+		if (!secondUserTag) secondUserTag = { avatar: null, discriminator: '0001', bot: false, username: '' };
```

Comment:
```suggestion
		const firstUserTag = this.client.userTags.get(firstUser.id) ?? { avatar: null, discriminator: '0001', bot: false, username: '' };
		const secondUserTag = this.client.userTags.get(secondUser.id) ?? { avatar: null, discriminator: '0001', bot: false, username: '' };
```

Reply:
Or even better:

```ts
private readonly kEmptyUserTag = { avatar: null, discriminator: '0001', bot: false, username: '' };
```

And then:

```suggestion
		const firstUserTag = this.client.userTags.get(firstUser.id) ?? this.kEmptyUserTag;
		const secondUserTag = this.client.userTags.get(secondUser.id) ?? this.kEmptyUserTag;
```